include/battle_scripts.h

```diff
@@ -456,6 +456,8 @@ extern const u8 BattleScript_HealingWishActivates[];
 extern const u8 BattleScript_LunarDanceActivates[];
 extern const u8 BattleScript_ShellTrapSetUp[];
 extern const u8 BattleScript_CouldntFullyProtect[];
+extern const u8 BattleScript_StealthRockActivates[];
+extern const u8 BattleScript_SpikesActivate[];
```

Comment:
BattleScript_SpikesActivates

Reply:
Why? "The Spikes [as in they] *activate*" 🤔 ...

Reply:
I guess you could also say that "the side status known as Spikes activates"...?

Sorry, I just didn't think about it like that lol. I'll make the change.

I already made the changes to `BattleScript_EffectHitSetEntryHazard` and the `case MOVE_EFFECT_DIRE_CLAW` of `Cmd_setmoveeffect` fwiw. I was just waiting for your response in the comments for `BattleScript_SpikesActivate` and `activateAfterFaint`.